cpp/example_code/s3/README.md

```diff
@@ -77,15 +84,9 @@ Code examples that show you how to accomplish a specific task by calling multipl
 functions within the same service.
 
 - [Create a presigned URL](presigned_get_object.cpp)
-- [Get started with buckets and objects](s3_getting_started_scenario.cpp)
+- [Create a serverless application to manage photos](cpp/example_code/cross-service/photo_asset_manager)
```

Comment:
Is this path correct?

Reply:
Fixed WRITEME to generate the right paths for these.